### PR TITLE
fix issues in vue template

### PR DESCRIPTION
**Pull request by personball**
fix some issues in vue template:

1. Fix css in main.less when build for production
2. Fix vue.config.js, files should be copy into dist folder when build for production
3. Fix appconst.ts defaultLocalizationSourceName should be AbpProjectName
4. Sample code about pagerequest of request a list in user.vue role.vue tenant.vue has a sql injection problem, fix user.vue and add todo for role.vue tenant.vue, also, add console.warn in Util.buildFilters
5. Replace PagedResultRequest with PagedUserResultRequest in IUserAppService as sample code deal with query filters.
6. Add rename.ps1 for test

### CODE REVIEW

**Comment by alirizaadiyahsi** [comment]
Hi @personball, thanks for contribution. But I can't understand why you added rename.ps1 ? Why we need this?

**Comment by personball** [comment]
Hi @alirizaadiyahsi, with rename.ps1, we can test if some where in project template missing a placeholder. For example, appconst.ts in vue\src\lib\ , has a defaultLocalizationSourceName  "FengCloud"  before.

Add rename.ps1 here just for test purpose.
You can remove it as your please.

**Comment by alirizaadiyahsi** [comment]
Thanks @personball

File: vue/src/views/main.less [hunk 1]
```diff
@@ -43,7 +43,7 @@
     transition: all 3s;
 }
 .main{
-    position: absolute;
+    //position: absolute;
```
Comment (ryancyq) [inline vue/src/views/main.less]: should we remove this entirely?

**Pull request merged**